src/components/base/FormattedVal/index.js

```diff
@@ -114,7 +121,7 @@ function FormattedVal(props: Props) {
   })
 
   return (
-    <T color={color || marketColor} withIcon={withIcon} {...p}>
+    <T color={color || marketColor} withIcon={withIcon} {...p} style={ellipsisStyles}>
```

Comment:
```suggestion
    <T color={color || marketColor} withIcon={withIcon} style={ellipsisStyles} {...p}>
```

in case at some place we were overriding the style we shouldn't break this possibility

also, shouldn't this be done in `T`'s style itself?

Reply:
Moved the styles to T